code/modules/antagonists/paradox_clone/paradox_clone.dm

```diff
@@ -0,0 +1,116 @@
+/datum/antagonist/paradox_clone
+	name = "\improper Paradox Clone"
+	roundend_category = "Paradox Clone"
+	job_rank = ROLE_PARADOX_CLONE
+	antag_hud_name = "paradox_clone"
+	suicide_cry = "THERE CAN BE ONLY ONE!!"
+	preview_outfit = /datum/outfit/paradox_clone
+	count_against_dynamic_roll_chance = TRUE
+
+	///Weakref to the clone's original, the target.
+	var/datum/weakref/original_ref
+
+/datum/antagonist/paradox_clone/get_preview_icon()
+	var/icon/final_icon = render_preview_outfit(preview_outfit)
+
+	final_icon.Blend(make_background_clone_icon(preview_outfit), ICON_UNDERLAY, -8, 0)
+
+	final_icon.Scale(64, 64)
+
+	return finish_preview_icon(final_icon)
+
+/datum/antagonist/paradox_clone/proc/make_background_clone_icon(datum/outfit/clone_fit)
+	var/mob/living/carbon/human/dummy/consistent/clone = new
+
+	var/icon/clone_icon = render_preview_outfit(clone_fit, clone)
+	clone_icon.ChangeOpacity(0.5)
+	qdel(clone)
+
+	return clone_icon
+
+/datum/outfit/paradox_clone
+	name = "Paradox Clone (Preview only)"
+
+	uniform = /obj/item/clothing/under/rank/civilian/janitor
+	gloves = /obj/item/clothing/gloves/color/black
+	head = /obj/item/clothing/head/soft/purple
+
+/datum/antagonist/paradox_clone/on_gain()
+	forge_objectives()
+	return ..()
+
+/datum/antagonist/paradox_clone/Destroy()
+	original_ref = null
+	return ..()
+
+/datum/antagonist/paradox_clone/proc/forge_objectives()
+
+	if(!original_ref)//admins didn't set one
+		original_ref = WEAKREF(find_original())
+	if(!original_ref)//we didn't find one
+		stack_trace("[src] was unable to find a target.")
+		qdel(src)
+		return
+
+	var/datum/objective/assassinate/paradox_clone/kill = new
+	kill.owner = owner
+	kill.target = original_ref.resolve()
+	kill.update_explanation_text()
+	objectives += kill
+
+/datum/antagonist/paradox_clone/proc/find_original()
+	var/list/viable_minds = list()
+	var/list/possible_targets = list() //filters out silicons and simplemobs
+	var/chosen_victim  //The cloned player
+
+	for(var/mob/living/carbon/human/player as anything in GLOB.player_list)//prevents cloning of non-crew
```

Comment:
```suggestion
	for(var/mob/living/carbon/human/player in GLOB.player_list)//prevents cloning of non-crew
```

I forgot to put this in my suggestion (it will make it actually check player_list for human)